src/component/footer/footer.js

```diff
@@ -2,22 +2,16 @@ import React from 'react';
 import CacheButton from './cache.js';
 import Throttle from './throttle.js';
 import RequestCount from './request-count.js';
+import ProxyStatus from './proxy-status';
```

Comment:
You should opt to use `.js` on your file imports, just in case things change down the road, wrt module spec.